src/screens/Settings/AboutUsScreen.tsx

```diff
@@ -30,16 +31,21 @@ export const AboutUsScreen = () => {
   };
 
   const handleRateButtonPress = () => {
+    const openAppStoreIfInAppFails = Platform.select({ android: true, ios: false }) as boolean;
+    const preferInApp = Platform.select({ android: false, ios: true }) as boolean;
```

Comment:
I guess forcing ` as boolean` is not necessary here.

Reply:
ok can be